lib/synced/timestamp.rb

```diff
@@ -0,0 +1,9 @@
+class Synced::Timestamp < ActiveRecord::Base
+  self.table_name = 'synced_timestamps'
+  belongs_to :parent_scope, polymorphic: true
+  scope :with_scope, ->(parent_scope) { where(parent_scope: parent_scope) }
+
+  def self.last_synced_at
+    maximum(:synced_at)
```

Comment:
Wouldn't be faster to `order(id: desc).first && order(id: desc).first.synced_at`? I'm quite curious, never seen any benchmark like this.